src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -48,13 +48,18 @@
 .monaco-workbench .pane-body.integrated-terminal .xterm {
 	/* All terminals have at least 10px left/right edge padding and 2 padding on the bottom (so underscores on last line are visible */
 	padding: 0 10px 2px;
-	/* Bottom align the terminal withing the split pane */
+	/* Bottom align the terminal within the split pane */
 	position: absolute;
 	bottom: 0;
 	left: 0;
 	right: 0;
 }
 
+.view-is-vertical .terminal.xterm {
```

Comment:
Is "vertical view" the right vernacular for this?

Reply:
maybe `view-in-side-panel` would be better considering the action is called `move views to side panel`?

Reply:
I think this will also be when the terminal lives in the side bar as well:

![image](https://user-images.githubusercontent.com/2193314/152071048-e6a90b01-2b02-45e3-b1e0-288fccb5ef75.png)

Not a big deal, we should try align the wording with however the view service refers to them

Reply:
ok will look into that. just tested and works well there too

Reply:
doesn't look like others have special styling for when it's vertical vs horizontal. 

since side bar and side panel have side in common, i'm thinking to go with `terminal-side-view`